repo/js/AutoFriendshipFight/README.md

```
@@ -101,9 +101,9 @@
     - 单次战斗超时时间（默认120秒）
     - 战斗超时会直接取消任务，因此不建议过短
     - 脚本会自动检测任务结束，识别精度很高
-   - 禁用异步战斗（更稳定；需配置“战斗结束检测”；对盗宝团无效）
-    - 适用于愚人众/鳄鱼等：依赖“自动检测战斗结束”退出战斗
-    - 盗宝团：仍使用路径战斗 + OCR 判定流程
+  - 禁用异步战斗（更稳定；需配置“战斗结束检测”；对所有敌人生效）
```

markdownlint-cli2 (0.22.1)

[warning] Inconsistent indentation for list items at the same level
Expected: 3; Actual: 2

(MD005, list-indent)